transit_odp/avl/tasks.py

```diff
@@ -116,7 +116,7 @@ def task_validate_avl_feed(task_id: str):
 
 @shared_task(bind=True)
 def task_create_sirivm_zipfile(self):
-    URL = f"{settings.CAVL_CONSUMER_URL}/datafeed"
+    URL = f"{settings.CAVL_API_BASE_URL}/siri-vm"
```

Comment:
Do these changes need a feature flag, in case bods does a release before our service is ready?

Reply:
by changes I mean this line and the changed line further down